.prettierrc

```diff
@@ -1,5 +1,5 @@
 useTabs: true
-tabWidth: 2
+tabWidth: 4
```

Comment:
@Aljullu take note I changed this in this pull because the `tabWidth` configuration what is set in the `.editorconfig`.  This meant that prettier was not correctly fixing lines going over the 80 char limit because it was configured for a smaller tabWidth.

I decided to slip this in this pull because it affected a file being edited and this is a smaller pull that will likely get merged soon anyways.

Reply:
Oh, right. Good catch with this too. I think we will need to run `npm run reformat-files` again so all files get properly formatted.

Reply:
If you want to do that I'm fine with it being done and merged into master.